tests/chocolatey-tests/choco-upgrade.Tests.ps1

```diff
@@ -0,0 +1,122 @@
+Import-Module helpers/common-helpers
+
+Describe "choco upgrade" -Tag Chocolatey, UpgradeCommand {
+    BeforeAll {
+        Initialize-ChocolateyTestInstall
+
+        New-ChocolateyInstallSnapshot
+    }
+
+    AfterAll {
+        Remove-ChocolateyTestInstall
+    }
+
+    Context "Upgrading a side-by-side Package (non-existing)" {

```

Comment:
Could we skip this if version is below 1.2.0-alpha? That will allow these tests to be bypassed with current stable builds of Chocolatey.